Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -39,10 +39,30 @@ class MultilineFormatter: PrettyFormatter {
         if fields.count == 1, let field = fields.first {
             body = "\(field.0): \(field.1)"
         } else {
-            let contents = fields.map { "\($0): \($1)" }.joined(separator: ",\n")
-            body = contents.indentTail(size: prefix.count)
+            body = fields.map(indentInsertedKeyValueString(_:))
```

Comment:
May I it make inlining? 😅 

e.g.

```swift
body = fields
    .map { label, value in "\(label): \(value.indentTail(size: "\(label): ".count))" }
    .joined(separator: ",\n")
    .indentTail(size: prefix.count)
```

Reply:
Sure.

Reply:
I make `indentInsertedKeyValueString()` inline. 5a2fa8a